frontend/simple/js/state.js

```diff
@@ -28,8 +29,10 @@ const state = {
   currentGroupId: null,
   contracts: {}, // contractIds => { type:string, recentHash:string } (for contracts we've successfully subscribed to)
   pending: [], // contractIds we've just published but haven't received back yet
-  loggedIn: false // false | { name: string, identityContractId: string }
+  loggedIn: false, // false | { name: string, identityContractId: string }
+  handlingEvent: false
 }
+var loggingOut = false
```

Comment:
👎 

There has to be a better way of handling this. Can't have variables for every single edge case that comes up. This leads to spaghetti code.